### DISCUSSION TITLE
Make AppImages Updateable

### DISCUSSION THREAD

**Discussion by Ultrabenosaurus** [ente-io/ente#4331] [Enhancements]
I would like to suggest that the AppImage releases of Ente desktop software for linux be made updateable, per the AppImage spec:

https://docs.appimage.org/packaging-guide/optional/updates.html

This would allow users to receive updates without having to re-download the entire program, and would also allow the AppImages to update themselves internally.

**Comment by mak448a** [comment]
I'm going to close #4117 since it's the same thing as this discussion
